packages/fluentui/docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -205,7 +205,14 @@ class ComponentDoc extends React.Component<ComponentDocProps, ComponentDocState>
                 key={info.displayName}
                 style={{ marginTop: '1rem' }}
               />
-              <Grid columns="auto 300px" styles={{ justifyContent: 'normal', justifyItems: 'stretch' }}>
+              <Grid
+                styles={{
+                  gridTemplateColumns: 'auto 300px',
+                  msGridColumns: `auto 300px`,
+                  justifyContent: 'normal',
```

Comment:
As I remember it's not supported in IE11 for CSS Grid

Reply:
add `msFlexPack: 'normal',`